src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/Stream.Collections.cs

```diff
@@ -14,19 +14,19 @@
 
 namespace System.Text.Json.Serialization.Tests
 {
-    public static partial class StreamTests
+    public partial class StreamTests
```

Comment:
Should this be `abstract` as well?

Reply:
It doesn't need to be since that was declared elsewhere. However, I'll make a new StreamTests.cs and declaring it there.